go/memcached-operator/Makefile

```diff
@@ -58,11 +58,12 @@ code-gen: ## Run the operator-sdk commands to generated code (k8s and openapi)
 test-e2e: ## Run integration e2e tests with different options.
 	@echo ... Running the same e2e tests with different args ...
 	@echo ... Running locally ...
+	- kubectl create namespace ${NAMESPACE} || true
 	- operator-sdk test local ./test/e2e --up-local --namespace=${NAMESPACE}
```

Comment:
Why the true is required here?

Reply:
**if without true, there is error reported.**
```
memcached-operator clyang$ make test-e2e
... Running the same e2e tests with different args ...
... Running locally ...
kubectl create namespace memcached
Error from server (AlreadyExists): namespaces "memcached" already exists
make: [test-e2e] Error 1 (ignored)
operator-sdk test local ./test/e2e --up-local --namespace=memcached
INFO[0000] Testing operator locally.
```
**if has true**
```
memcached-operator clyang$ make test-e2e
... Running the same e2e tests with different args ...
... Running locally ...
kubectl create namespace memcached || true
Error from server (AlreadyExists): namespaces "memcached" already exists
operator-sdk test local ./test/e2e --up-local --namespace=memcached
INFO[0000] Testing operator locally.
```